tests/unit/test_clustering.py

```diff
@@ -9,8 +9,27 @@ def test_kmeans(test_client, test_sample_vector_dataset):
         overwrite=True,
     )
     db_health = test_client.datasets.monitor.health(test_sample_vector_dataset)
+    test_client.vector_tools.cluster.plot_clusters(
```

Comment:
When you write tests, you should not implement them all into 1. In this particular section here, you should split up your tests so that each test tests only 1 thing.